scripts/ci/setup-ipfs.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -e
+
+echo "::group::Install go-ipfs and ipfs-cluster-ctl"
+    curl -s https://dist.ipfs.io/go-ipfs/${GO_IPFS_VER}/go-ipfs_${GO_IPFS_VER}_linux-amd64.tar.gz | sudo tar vzx -C /usr/local/bin/ go-ipfs/ipfs --strip-components=1
+    curl -s https://dist.ipfs.io/ipfs-cluster-ctl/${CLUSTER_CTL_VER}/ipfs-cluster-ctl_${CLUSTER_CTL_VER}_linux-amd64.tar.gz | sudo tar vzx -C /usr/local/bin/ ipfs-cluster-ctl/ipfs-cluster-ctl --strip-components=1
+echo "::endgroup::"
+
+# fix resolv - DNS provided by Github is unreliable for DNSLik/dnsaddr
+sudo sed -i -e 's/nameserver 127.0.0.*/nameserver 1.1.1.1/g' /etc/resolv.conf
```

Comment:
:information_source: This was pretty annoying. Entire build failed because Githubs DNS proxying cache at `127.0.0.*`  randomly timeouted while resolving `_dnslink` or `_dnsaddr`. Switching to Cloudflare here make the problem go away.